lib/google_contacts_api/result.rb

```diff
@@ -22,6 +21,12 @@ def id
       _id ? _id["$t"] : nil
     end
 
+    # Item path relative to Google contacts URL base for puts/posts/deletes
+    def id_path
```

Comment:
Good question. I think perhaps as we eventually have more Group/Contact mutation functionality (create/edit/delete) then it may become more clear what the role of this method is and where it should go. I think in my fork of the gem I use it for both contacts and groups since they follow a similar URL naming scheme in that it should have `full` in the URL for creates/updates but the "get" calls return the URL with `base`.